packages/lit-dev-content/site/blog/2022-10-18-lit-labs-event.md

```diff
@@ -0,0 +1,38 @@
+---
+tags: blogPosts
+layout: blog-post.html
+title: "Get the low-down on Lit Labs"
+summary: "Join us for a virtual event on October 24th 2022 at 10AM PDT to learn all about Lit Labs!"
+date: 2022-10-18
+author:
+  - elliott-marquez
+eventDate: "Monday, October 24th 2022 at 10AM PDT"
+ytLink: https://www.youtube.com/watch?v=l6Gn5uV83sw
+---
+
+The Lit team is hosting a virtual event on **{{eventDate}}** to talk about all of the exciting work going on in Lit Labs. We'll give an overview of the Lit Labs program, followed by detailed presentations on several Labs packages. And we'll let you know how you can help get these packages across the finish line and shape the future of Lit!
+
+To make sure you don't miss it, [**go to the live stream**]({{ytLink}}) and sign up for notifications.
+
+<img src="/images/blog/labs-event/lit-labs.svg" width="720" class="block centered">
+
+## What is Lit Labs?
+
+[Lit Labs]({{site.baseurl}}/docs/libraries/labs/) is an umbrella for projects related to Lit that aren't yet ready for production. These projects may be experimental or incomplete, or simply looking for real-world feedback. Lit Labs projects are published under the [`@lit-labs` npm scope](https://www.npmjs.com/search?q=%40lit-labs).
```

Comment:
This looks like text from the old Labs page that we workshopped a bit when doing the labs page refresh in https://github.com/lit/lit.dev/pull/930#pullrequestreview-1099983917. 

Same as there, I'd prefer to not say "not ready for production" and instead use the [revised words](https://lit.dev/docs/libraries/labs/) that don't accidentally discourage real-world usage/feedback.

```suggestion
[Lit Labs]({{site.baseurl}}/docs/libraries/labs/) is an umbrella for Lit packages under development that we are actively seeking feedback on. These projects may be experimental or incomplete, or simply looking for real-world feedback. Lit Labs projects are published under the [`@lit-labs` npm scope](https://www.npmjs.com/search?q=%40lit-labs).
```

The "feedback" part here might be a little duplicative, might need a little more massage.

Otherwise LGTM!